cypress/integration/examples/mainTest.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("main test", () => {
+  it("open main site and check console", () => {
+    cy.visit("http://localhost:8080")
```

Comment:
We will have to create environment.ts file for it so it doesn't need to be hardcoded.
I will add the task.

Reply:
ok